sql/core/src/test/java/test/org/apache/spark/sql/sources/v2/JavaSimpleBatchTable.java

```diff
@@ -52,53 +48,3 @@ public String name() {
   }
 }
 
-abstract class JavaSimpleScanBuilder implements ScanBuilder, Scan, Batch {
```

Comment:
These were moved to top-level classes. Multiple top-level classes in Java are mostly a no-no, and actually generate a warning if you try to use them in another file.